Char.php

```diff
@@ -1,275 +1,272 @@
 <?php
 
-$DFCharacters = array(
```

Comment:
Also, is there a requirement to move these into classes now, or is that just a stylistic preference?

Reply:
I encounter problem that cause null pointer error when access this array, i`m not php programmer so just move them in class where they will be guaranteed in function scope.

Reply:
this mentioned in commit 
Avoid using globals to fix: Color.php: PHP Notice: Trying to access array offset on value of type null \n Color.php: PHP Warning: Invalid argument supplied for foreach() on example

Reply:
Huh, yeah, could be related to a PHP upgrade or MW invoking code at a different time.